platforms/intellij/compose/playground/build.gradle.kts

```diff
@@ -47,6 +49,14 @@ kotlin {
         implementation(projects.platforms.intellij.compose)
       }
     }
+    jvmTest {
+      dependencies {
+        implementation(libs.junit)
+        implementation(libs.roborazzi)
+        implementation(compose.desktop.common)
+        implementation(compose.desktop.uiTestJUnit4)
```

Comment:
Would this also be the correct compose desktop test dependency to add?

Reply:
What does the sample in roborazzi use? Probably easiest to just match that to start

Reply:
It seems [here](https://github.com/takahirom/roborazzi/blob/main/roborazzi-compose-desktop/build.gradle) they don't have dependencies in `jvmTest`, but they have these in the `commonMain` dependencies:

```
implementation(compose.runtime)
implementation(compose.desktop.currentOs)
implementation(project(":roborazzi-painter"))
implementation(libs.kotlin.stdlib.jdk8)
api libs.compose.ui.test.junit4.desktop
implementation libs.compose.ui.graphics.desktop
```

Reply:
That's the library itself, we should look at one of their samples. I was eyeing this: https://github.com/takahirom/roborazzi/blob/main/sample-compose-desktop-jvm/build.gradle.kts